step-02/lib/product_row_item.dart

```diff
@@ -1,110 +1 @@
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
-import 'package:flutter/cupertino.dart';
-import 'package:scoped_model/scoped_model.dart';
-import 'model/app_state_model.dart';
-import 'model/product.dart';
-import 'styles.dart';
-
-class ProductRowItem extends StatelessWidget {
-  const ProductRowItem({
-    this.index,
-    this.product,
-    this.lastItem,
-  });
-
-  final Product product;
-  final int index;
-  final bool lastItem;
-
-  @override
-  Widget build(BuildContext context) {
-    final row = SafeArea(
-      top: false,
-      bottom: false,
-      minimum: const EdgeInsets.only(
-        left: 16,
-        top: 8,
-        bottom: 8,
-        right: 8,
-      ),
-      child: Row(
-        children: <Widget>[
-          ClipRRect(
-            borderRadius: BorderRadius.circular(4),
-            child: Image.asset(
-              product.assetName,
-              package: product.assetPackage,
-              fit: BoxFit.cover,
-              width: 76,
-              height: 76,
-            ),
-          ),
-          Expanded(
-            child: Padding(
-              padding: const EdgeInsets.symmetric(horizontal: 12),
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.start,
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: <Widget>[
-                  Text(
-                    product.name,
-                    style: Styles.productRowItemName,
-                  ),
-                  const Padding(padding: EdgeInsets.only(top: 8)),
-                  Text(
-                    '\$${product.price}',
-                    style: Styles.productRowItemPrice,
-                  )
-                ],
-              ),
-            ),
-          ),
-          CupertinoButton(
-            padding: EdgeInsets.zero,
-            child: const Icon(
-              CupertinoIcons.plus_circled,
-              semanticLabel: 'Add',
-            ),
-            onPressed: () {
-              final model = ScopedModel.of<AppStateModel>(context);
-              model.addProductToCart(product.id);
-            },
-          ),
-        ],
-      ),
-    );
-
-    if (lastItem) {
-      return row;
-    }
-
-    return Column(
-      children: <Widget>[
-        row,
-        Padding(
-          padding: const EdgeInsets.only(
-            left: 100,
-            right: 16,
-          ),
-          child: Container(
-            height: 1,
-            color: const Color(0xFFD9D9D9),
```

Comment:
Maybe just reference the styles file you already made

Reply:
Fixed in https://github.com/googlecodelabs/flutter-cupertino-store/pull/6/files#diff-e984079777bf36788a762b77ed2c1615